Mage/src/main/java/mage/cards/decks/importer/MtgaImporter.java

```diff
@@ -36,10 +36,11 @@ protected void readLine(String line, DeckCardLists deckList, FixedInfo fixedInfo
         line = line.trim();
 
         if (line.equals("Deck")) {
+            sideboard = false;
             return;
         }
 
-        if (line.equals("Sideboard") || line.equals("")) {
+        if (line.equals("Sideboard") || line.equals("") || line.equals("Commander")) {
```

Comment:
Can we also change this to case-insensitive comparison? (Same for the `"Deck"` check).